scripts/tests/test_downloads_summary.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+import mock
+import pytest
+import pytz
+import datetime
+
+from django.utils import timezone
+
+from addons.osfstorage import utils
+from addons.osfstorage.tests.utils import StorageTestCase
+
+from osf_tests.factories import ProjectFactory
+
+from scripts.analytics.download_count_summary import DownloadCountSummary
+
+
+@pytest.mark.django_db
+class TestDownloadCount(StorageTestCase):
+
+    def test_download_count(self):
+        # Keen does not allow same day requests so we have to do some time traveling to my birthday
+        timezone.now = mock.Mock(return_value=datetime.datetime(1991, 9, 25).replace(tzinfo=pytz.utc))
```

Comment:
Use `mock.patch` to temporarily change the return values of things rather than assigning the attribute. That way, you don't have to worry about re-setting the attribute when the test is finished.